translations/translation_utility.sh

```diff
@@ -42,10 +42,11 @@ function Confirmation() { # Process Yes/No/Quit response
 # Functions End
 
 # ===== Modules ====
+bin_path=""
 Check_bin() { # Check if the bin path exists
-    bin_exists=$(/usr/bin/env lupdate 2>/dev/null) 
+    bin_exists=$(/usr/bin/env lupdate 2>/dev/null)
     if [[ $bin_exists ]]; then
-        bin_path=$(dirname "$bin_exists")
+        bin_path=$(dirname "$(which lupdate)")
```

Comment:
why not `env`?

Reply:
Actually `bin_path=$(dirname "$bin_exists")` should work

Reply:
It won't work on a Mac. I've tested.

Reply:
What? This is the standard POSIX method, and as such should work on every UNIX like system

Reply:
I found the reason. I forgot to exit python and bash

Reply:
Actually they work.

Reply:
So why not using it?

Reply:
Apple has limited the use of `/usr/bin/env￼` on macOS, Mac users always use bash. That could be part the reason.

Reply:
I have dealt with this problem. But maybe I need more time.

Reply:
Additional instructions for Mac users have to be added to the markdown document￼. This will be done later. ￼